README.md

```diff
@@ -25,53 +38,56 @@ Use [Prettier](https://prettier.io/) to format markdown pages. To run Prettier o
 run:
 
 ```bash
-npx prettier --write ./docs/
+$ npx prettier --write ./docs/
```

Comment:
Adding `$` makes these commands un-copy-pasteable. Usually, you would put `$` when the code fence has some output as well, so as to differentiate between commands and outputs, but when the command is meant to be run verbatim, the `$` is in the way.